src/pages/dev/general-message-passing/gmp-messages.mdx

```diff
@@ -2,6 +2,16 @@
 
 import { Callout } from "/src/components/callout"
 
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/watch?v=htMVIYzGA34"
```

Comment:
this should be /embed/ (see https://forum.freecodecamp.org/t/youtube-refused-to-connect/245262)